### DISCUSSION TITLE
Clase - Backend

### DISCUSSION THREAD

**Discussion by GP2023J** [Launch-X-Latam/MisionFrontEnd#537] [Team Posts]
Hola a todos amigos, disculpen pero horita se está realizando la 2da clase del desafio Backend alguien me puede pasar el link? mandé mi solicitud para que me aparezca el grupo pero aún no me aceptan. Gracias.

**Comment by agenciakolob** [comment]
Todavia no ha empezado, este es el [link](https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZmM4MjZlOGItYjU5NC00NWQyLTk5ZmYtZGJkY2E0NjQzYjJl%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%22c3d76ba1-72de-4bf4-be64-d1c6a6bd4143%22%2c%22IsBroadcastMeeting%22%3atrue%7d)

**Comment by GP2023J** [comment]
> Todavia no ha empezado, este es el link

Muchas gracias amigo!!
